bql/semantic/semantic_test.go

```diff
@@ -259,6 +259,32 @@ func TestGraphClauseManipulation(t *testing.T) {
 	}
 }
 
+func TestFilterClauseManipulation(t *testing.T) {
+	st := &Statement{}
+
+	t.Run("test workingFilter initial states", func(t *testing.T) {
```

Comment:
this will be printed as:

TestFilterClauseManipulation/test_workingFilter_initial_states

so we can skip the word 'test'. Also a recommendation is to include the expected return for your test case like:
"workingFilter with initial states success" / or returns OK

Reply:
Done. Thank you for the tips Jose. =]